tests/src/Functional/UserSubscriptionsPageTest.php

```diff
@@ -73,4 +73,16 @@ public function testUserPreferences(): void {
     $this->doTestUserPreferences($this->drupalCreateUser(), $fn_get_path);
   }
 
+  /**
+   * Tests the form configuration.
+   */
+  public function testFormConfiguration(): void {
```

Comment:
The name is misleading, we are testing the "preface" of the form. Let's then explain in the comment what are we testing: the rendering of the introduction text and terms url.

Reply:
Changed